pivotal.py

```diff
@@ -0,0 +1,48 @@
+import scrapy
+import utils
+from datetime import datetime
+
+
+def serialize_date(value):
+    """
+    Serialize the string date to a datetime value.
+    :param value:
+    :return:
+    """
+    return datetime.strptime(value, '%d %b %Y')
+
+
+class PivotalSpider(scrapy.Spider):
+    utils.init_datadog()
+
+    name = "pivotal_security"
+    start_urls = ['https://pivotal.io/security']
+
+    def parse(self, response):
+        rows = response.css('table tr')
+
+        if len(rows) <= 0:
+            utils.Report.warning()
+
+        for cve in rows:
+            fields = cve.css('td')
+
+            try:
+                date = serialize_date(utils.get_string(fields[0].css('::text').extract()))
+            except ValueError:
+                continue
```

Comment:
We kind of expect the header to appear here. This isn't in fact, the actual check if there is something wrong with scrapy.